src/lib/factories.js

```diff
@@ -46,7 +46,7 @@ export function createShorthand(Component, mapValueToProps, val, defaultProps =
   } else if (_.isPlainObject(val)) {
     type = 'props'
     usersProps = val
-  } else if (_.isString(val) || _.isNumber(val)) {
+  } else if (_.isString(val) || _.isNumber(val) || _.isArray(val)) {
```

Comment:
(...submitting a little half baked, ran out of time on this comment but want to get the convo started)

I'm not sure we should pass arrays on to be mapped to props.  It means every component might do something different with the array (or not handle it at all).  If the factory does handle arrays, IMHO it is most intuitive to expect it to always create an array of elements.

```jsx
const button = Button.create('Submit')

// `key` derived from the string/number literal or `childKey` prop
const buttons = Button.create(['A', 123])
const buttons = Button.create([{ content: 'A' }, { content: 123 }])
const buttons = Button.create([<Other>A</Other>, <Other>123</Other>])
```

Then we can standardize shorthand propTypes since all components with shorthand would be capable of handling arrays.

If the factory passes it along, we'd need to make sure every `mapValueToProps` handles arrays.  Though, every component then might do something different with the array which isn't clear to the user.  "How does X handle arrays again?" Also, not all components / `mapValueToProps()` may be able support arrays.  It could make standard propTypes tough.

Lastly, when passing an array, you are creating an array of X components.  If there is a need to pass that array through the X's props to some Y's shorthand, then the first component is just a pass through.  My gut says there should be a "one level deep" design decision.  Meaning, we would forgo the middle man and pass the shorthand directly to the Y instead of through X to Y.

...totally open on this, just thinking out loud.